packages/core/src/connectors/utilities/index.ts

```diff
@@ -33,3 +37,29 @@ export function validateConnectorModule(
     throw new ConnectorError(ConnectorErrorCodes.UnexpectedType);
   }
 }
+
+export const readUrl = async (
+  url: string,
+  baseUrl: string,
+  type: 'text' | 'svg'
```

Comment:
I think we said we can also support other image formats, like png

Reply:
YES, we already have LOG-3957